### PR TITLE
Added west key

### PR DESCRIPTION
**Pull request by pablogs9**
Added west key for micro-ROS project

Zephyr RTOS build tool available via pip:
https://pypi.org/project/west/

### CODE REVIEW

File: rosdep/base.yaml
Comment (clalancette) [inline rosdep/base.yaml]: Since this is a pip-only key, you'll need to add the `-pip` prefix.  Also you'll need the `python` prefix; see https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1 for the pip rules.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/base.yaml
Comment (pablogs9) [reply rosdep/base.yaml]: Done, thanks!

Comment (clalancette) [reply rosdep/base.yaml]: This is failing CI since it needs to be in alphabetical order now.

Comment (pablogs9) [reply rosdep/base.yaml]: Sorry, done!

Comment (clalancette) [reply rosdep/base.yaml]: I'm sorry I didn't notice before, but this is also in the wrong file.  Python keys should go into https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml .  Once its moved there, I'm happy to approve.

Comment (pablogs9) [reply rosdep/base.yaml]: Done! Thanks

**Review by clalancette — APPROVED** [review]

**Comment by pablogs9** [comment]
Do I need a second reviewer in order to merge?

**Comment by pablogs9** [comment]
I can't add reviewers, would you mind to add some other?
Thanks.

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**
